src/app.js

```diff
@@ -86,12 +56,15 @@ app.use(compress())
 	.configure(requestLogger)
 	.use((req, res, next) => {
 		// pass header into hooks.params
+		const uid = ObjectId();
+		req.headers.requestId = uid.toString();
```

Comment:
only set the requestId, if it is not defined, then we can already set it inside the client and reuse it here

Reply:
In client i do not want to add it. But if the API gateway is established, then we can remove this code. I mark it as temp solution.

Reply:
Is marked.